src/main/scala/cognite/spark/v1/SdkV1Relation.scala

```diff
@@ -61,7 +61,7 @@ abstract class SdkV1Relation[A <: Product, I](config: RelationConfig, shortName:
       updates: Seq[P],
       resource: T,
       isUpdateEmpty: U => Boolean
-  )(implicit transform: Transformer[P, U]): IO[Unit] = {
+  )(implicit transform: Transformer.AutoDerived[P, U]): IO[Unit] = {
```

Comment:
`Transformer.AutoDerived` is a parent of `Transformer`, that should be used where auto derivation should be triggered in the absence of user-provided implicit.